packages/story-editor/src/components/canvas/karma/carousel.karma.js

```diff
@@ -33,6 +37,7 @@ describe('Carousel integration', () => {
 
   beforeEach(async () => {
     fixture = new Fixture();
+    elementTypes.forEach(registerElementType);
```

Comment:
🤔  is this something we could add to the fixture (`web-stories-wp/packages/story-editor/src/karma/fixture/fixture.js`) so that if it's needed again and nobody remembers this gotcha it's easier to troubleshoot? Would also centralize the imports.

Reply:
Oh yes 😬, thanks